src/mysky/index.ts

```diff
@@ -558,4 +667,8 @@ export class MySky {
   protected async signRegistryEntry(entry: RegistryEntry, path: string): Promise<Signature> {
     return await this.connector.connection.remoteHandle().call("signRegistryEntry", entry, path);
   }
```

Comment:
do not await returned value, it's already a promise

```suggestion
  protected signRegistryEntry(entry: RegistryEntry, path: string): Promise<Signature> {
    return this.connector.connection.remoteHandle().call("signRegistryEntry", entry, path);
  }
```

Reply:
https://stackoverflow.com/questions/38708550/difference-between-return-await-promise-and-return-promise

Reply:
The link you pasted enforces the request to change it, I'm not sure why you gave it thumbs down?

Reply:
https://eslint.org/docs/rules/no-return-await

Reply:
> If you want the functions to show up in stack traces (useful for debugging purposes)

This is why me and @peterjan decided to start doing it this way. Not adding the lint because it seems subjective and I don't want to get into a discussion like that here.

Reply:
Oh, lol I just commented on this in the other comment without seeing this one 😂  As mentioned before, I am in the `return await` camp and do it religiously. You should try it, it actually even makes the code more reasonable and I have gotten into a situation before where it proved useful. That's actually why I knew about the call stack thing.